nf_core/__main__.py

```diff
@@ -1622,6 +2014,51 @@ def bump_version(new_version, dir, nextflow):
         sys.exit(1)
 
 
+# nf-core create-logo
+
+
+@nf_core_cli.command("create-logo")
+@click.argument("logo-text", metavar="<logo_text>")
+@click.option("-d", "--dir", type=click.Path(), default=".", help="Directory to save the logo in.")
+@click.option("-n", "--name", type=str, help="Name of the output file (with or without '.png' suffix).")
+@click.option(
+    "--theme",
+    type=click.Choice(["light", "dark"]),
+    default="light",
+    help="Theme for the logo.",
+    show_default=True,
+)
+@click.option(
+    "--width",
+    type=int,
+    default=2300,
+    help="Width of the logo in pixels.",
+    show_default=True,
+)
+@click.option(
+    "-f",
+    "--force",
+    is_flag=True,
+    default=False,
+    help="Overwrite any files if they already exist",
+)
+def logo(logo_text, dir, name, theme, width, force):
+    """
+    Generate a logo with the nf-core logo template.
+
+    This command generates a logo for your a given text <logo_text> underneath the 'nf-core/'. The text is usually the name of a pipeline."
+    """
+    from nf_core.create_logo import create_logo
+
+    try:
+        if dir == ".":
+            dir = Path.cwd()
+        create_logo(logo_text, dir, name, theme, width, force)
```

Comment:
We should log something so that the end user knows it worked.

```suggestion
        logo_path = create_logo(logo_text, dir, name, theme, width, force)
        log.info(f"Created logo: [magenta]{logo_path}[/]")
```